### DISCUSSION TITLE
[Proposal] Group and hide styles

### DISCUSSION THREAD

**Discussion by mosh-tudor** [GrapesJS/grapesjs#5988] [Plugin Suggestions]
## Proposal:
Add the ability to group (and hide) specific styles from the **Style Manager**.

Maybe something like this:

```js
  styleManager: {
    sectors: [...],
    groups: [{
      name: 'More options',
      open: false,
      sectors: [...],
    }, {
      name: 'Even more options',
      open: false,
      sectors: [...],
    }],
  },
```

## Why?
This will allow us to group styles more compactly & logical // without having a mess of random options.

## Example (from Webflow):
<img width="294" alt="Screenshot 2024-07-03 at 18 01 13" src="https://github.com/GrapesJS/grapesjs/assets/668451/b807a4ac-9d4d-4849-b46f-35402d62275d">

**Comment by glaprida** [comment]
I'm all in for this; though ideally not in the form of a plugin. Any chance to get this built-in?
